pkg/services/ngalert/api/compat_silences.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+// Silence-specific compat functions to convert between API and model types.
+
+func GettableSilenceToAPIGettableSilence(s models.Silence) definitions.GettableSilence {
```

Comment:
These function names are very confusing, WDYT about `SilenceToGettableSilence` or something similar? The Postable/Gettable prefixes should probably stay solely for API models.

Reply:
Ah good catch, these were named from an old version before I had the model struct and it shows :grimacing: